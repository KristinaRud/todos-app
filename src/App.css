.container {
  width: 100%;
  padding-right: calc(16px * 0.5);
  padding-left: calc(16px * 0.5);
  margin-right: auto;
  margin-left: auto;
}

@media (min-width: 576px) {
  .container {
    max-width: 540px;
  }
}

@media (min-width: 768px) {
  .container {
    max-width: 720px;
  }
}

@media (min-width: 992px) {
  .container {
    max-width: 960px;
  }
}

@media (min-width: 1200px) {
  .container {
    max-width: 1140px;
  }
}

@media (min-width: 1400px) {
  .container {
    max-width: 1320px;
  }
}

.page__wrapper {
	padding: 32px;
	box-sizing: border-box;
}

.page__dashboard {
	padding: 32px 40px;
	box-sizing: border-box;
	position: relative;
	min-height: 505px;
	height: 100%;
	border-radius: 4px;
	box-shadow: 0 8px 16px 0 rgb(64 76 79 / 16%);
}

