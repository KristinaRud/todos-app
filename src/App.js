import Dashboard from "./pages/Dashboard";
import Header from "./components/Header";
import AddTask from "./components/AddTask";
import ListTask from "./components/ListTask";
import Footer from "./components/Footer";

import './App.css';

function App() {

  return (
        <Dashboard>
          <Header/>
          <AddTask/>
          <ListTask/>
          <Footer/>
        </Dashboard>
  );
}

export default App;
