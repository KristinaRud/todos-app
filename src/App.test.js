import {render} from '@testing-library/react'
import App from "./App";
import ToDoProvider from "./context";


describe('snapshot App', ()=>{
    test('snapshot App', ()=>{

        const app = render(
            <ToDoProvider>
            <App/>
            </ToDoProvider>);
        expect(app).toMatchSnapshot();
    })
})
