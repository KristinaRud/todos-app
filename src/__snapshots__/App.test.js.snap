// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot App snapshot App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dashboard"
      >
        <div
          class="dashboard-container"
        >
          <header
            class="g-header"
          >
            <h1>
              Task Manager
            </h1>
            <div
              class="header-desc"
            >
              <p>
                Процент не выполненых задач 
                NaN
                %
              </p>
              <p>
                Количество созданных задач за сессию: 
                0
              </p>
            </div>
          </header>
          <div
            class="add-task"
          >
            <form
              class="form-task"
            >
              <input
                class="task-massage"
                maxlength="50"
                placeholder="Description of my new task"
                type="text"
              />
              <button
                class="btn task-btn"
                type="submit"
              >
                Добавить задачу
              </button>
            </form>
          </div>
          <div
            class="list-wrapper"
          >
            <ul
              class="list-tasks"
            />
          </div>
          <footer
            class="g-footer"
          >
            <label
              class="g-checkbox"
            >
              <input
                checked=""
                type="checkbox"
              />
              <div
                class="label"
              >
                <div
                  class="label-content"
                >
                  <span>
                    Все задачи выполнены
                  </span>
                </div>
              </div>
            </label>
          </footer>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dashboard"
    >
      <div
        class="dashboard-container"
      >
        <header
          class="g-header"
        >
          <h1>
            Task Manager
          </h1>
          <div
            class="header-desc"
          >
            <p>
              Процент не выполненых задач 
              NaN
              %
            </p>
            <p>
              Количество созданных задач за сессию: 
              0
            </p>
          </div>
        </header>
        <div
          class="add-task"
        >
          <form
            class="form-task"
          >
            <input
              class="task-massage"
              maxlength="50"
              placeholder="Description of my new task"
              type="text"
            />
            <button
              class="btn task-btn"
              type="submit"
            >
              Добавить задачу
            </button>
          </form>
        </div>
        <div
          class="list-wrapper"
        >
          <ul
            class="list-tasks"
          />
        </div>
        <footer
          class="g-footer"
        >
          <label
            class="g-checkbox"
          >
            <input
              checked=""
              type="checkbox"
            />
            <div
              class="label"
            >
              <div
                class="label-content"
              >
                <span>
                  Все задачи выполнены
                </span>
              </div>
            </div>
          </label>
        </footer>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
